### ISSUE TITLE
Motherboard Onboard Preferences does not open

### ISSUE THREAD

**Issue by rickarddahlstrand** [makerbot/ReplicatorG#226]
What steps will reproduce the problem?
1. Open ReplicatorG
2. Connect to Makerbot TOM
3. Select Motherboard Preferences

What is the expected output? What do you see instead?
The dialog should open, but nothing happens.

What version of the product are you using? On what operating system?
Mac OS 10.7 Lion.
ReplicatorG25.

Please provide any additional information below.
Other people have reported this and it seems like it's been reported before like here
http://wiki.makerbot.com/forum/t-371446

On ReplicatorG24 it works perfectly.

**Comment by phooky** [comment]
Adding this to the queue for 26.

**Comment by rickarddahlstrand** [comment]
Thanks!!

**Comment by eried** [comment]
I posted a fixed file for this issue in my page: http://servicios.ried.cl/makerbot/

Also I sent a message to makerbot support, the issue was in the MachineOnboardParameters of ReplicatorG code, the function roundDouble should detect the regional options (dot or comma) that's why probably 50% of the people experience the problem only.

**Comment by Stonehoof** [comment]
Hi all

Eried, could you show me patch you made to sources. Seems that there is another issue related to roundDouble
function
All values I put in MachineOnboardParameters  dialog fields are rounded. Floating part just discarded.
Using patched version of RepG 27 on Windows7 with Russian regiobal settings.

Tnx!

**Comment by eried** [comment]
Hi, I did not patched the sources (I hate a bit java haha, so I don't want to install the gazillion things related to java), I just edited the bytecode of the compiled class.

**Comment by Stonehoof** [comment]
Seems I need to do that then :)

**Comment by Stonehoof** [comment]
Seems the issue is solved for 027 beta

**Comment by eried** [comment]
Nice!! (Anyway I sent the details for the bug & fix, my patched file was only a very quick workaround)

**Comment by FarMcKon** [comment]
Yeah, Ted B fixed/tweaked all of the decimal and internationalization for input dialogs. This should be no problem going forward.
